commit ae86dd298706b20d58200a49d04d56bd8c1b0419
Author: Eric Lippmann <eric.lippmann@icinga.com>
Date:   Wed Nov 29 14:31:18 2017 +0100

    FPM Config: Support Apache < 2.4

diff --git a/packages/files/apache/icingaweb2.fpm.conf b/packages/files/apache/icingaweb2.fpm.conf
index fce026e45..9f01c214b 100644
--- a/packages/files/apache/icingaweb2.fpm.conf
+++ b/packages/files/apache/icingaweb2.fpm.conf
@@ -1,5 +1,12 @@
 Alias /icingaweb2 "/usr/share/icingaweb2/public"
 
+<IfVersion < 2.4>
+    # Forward PHP requests to FPM
+    <LocationMatch "^/icingaweb2/(.*\.php)$">
+        ProxyPassMatch "fcgi://127.0.0.1:9000/usr/share/icingaweb2/public/$1"
+    </LocationMatch>
+</IfVersion>
+
 <Directory "/usr/share/icingaweb2/public">
     Options SymLinksIfOwnerMatch
     AllowOverride None
@@ -38,9 +45,11 @@ Alias /icingaweb2 "/usr/share/icingaweb2/public"
         ErrorDocument 404 /icingaweb2/error_norewrite.html
     </IfModule>
 
-    # Forward PHP requests to FPM
-    <FilesMatch "\.php$">
-        SetHandler "proxy:fcgi://127.0.0.1:9000"
-        ErrorDocument 503 /icingaweb2/error_unavailable.html
-    </FilesMatch>
+    <IfVersion >= 2.4>
+        # Forward PHP requests to FPM
+        <FilesMatch "\.php$">
+            SetHandler "proxy:fcgi://127.0.0.1:9000"
+            ErrorDocument 503 /icingaweb2/error_unavailable.html
+        </FilesMatch>
+    </IfVersion>
 </Directory>
